updating readme, deleting orphaned changesets (#153)
RooVetGit/Roo-Cline: CHANGELOG.md

# Roo Cline Changelog

## [2.2.16]

> -   Incorporate dbasclpy's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2)

Reviewer: ```suggestion
-   Incorporate Premshay's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2)
```

> -   Incorporate dbasclpy's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2)
> -   Unified Bedrock calls using BedrockClient and Bedrock Runtime API from [PR](https://github.com/RooVetGit/Roo-Cline/pull/60)

Reviewer: ```suggestion
-   Incorporate Premshay's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2) and unify calls using BedrockClient and Bedrock Runtime API
```

# Roo Cline Changelog

## [2.2.16]

> -   Incorporate dbasclpy's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2) and unified Bedrock calls using BedrockClient and Bedrock Runtime API

Reviewer: ```suggestion
-   Incorporate Premshay's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2) and unified Bedrock calls using BedrockClient and Bedrock Runtime API
```
